client/src/components/Assets/Icons/AccountCircleIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { IconProps } from ".";
+
+const AccountCircleIcon = ({
+  className = "",
+  width = "20",
+  height = "20",
+  fill
+}: IconProps) => (
+  <svg
+    className={className}
+    width={width}
+    height={height}
+    viewBox="0 0 20 20"
+    fill="none"
```

Comment:
is this a different `fill` than the one from props?

Reply:
Yup we want to fill the path and not the svg tag